model/riscv_pmp_regs.sail

```diff
@@ -216,3 +244,15 @@ function pmpWriteAddr(locked: bool, tor_locked: bool, reg: xlenbits, v: xlenbits
   if   sizeof(xlen) == 32
   then { if (locked | tor_locked) then reg else v }
   else { if (locked | tor_locked) then reg else EXTZ(v[53..0]) }
+
+function mseccfgWrite(reg: xlenbits, v: xlenbits) -> xlenbits =   /* to set RLB, need to check PMPCFG_L */
+  if mseccfg.RLB() == 0b0 & v[2] == bitone then {
```

Comment:
As with checkExe, minimise opaque bit vector indexing

Reply:
it would hinder the readability I think this should be ok...as  simple to read

Reply:
No it wouldn't. As it stands nobody reading this code has any clue what these magic sets of bits are. The whole point of the types is so that these magic bits get names that the code uses everywhere. You should only be using bitvectors when necessary, and never accessing bits within them directly.